generationkwh/usagetracker.py

```diff
@@ -36,9 +38,12 @@ def use_kwh(self, member, start, end, fare, period, kwh):
             used = min(kwh-allocated, p-u)
             usage[i] += used
             allocated += used
+            if kwh - allocated < 1:
+                break
 
         self._usage.updateUsage(member, start, usage)
-        return allocated
+        usage_date_dict = self.convert_usage_date_quantity(usage, start, end)
```

Comment:
What about the tests?

Reply:
Suggestion for the transition:
- Rename the method
- Write a wrapper use_kwh that returns the old value calling the renamed method
- Use the renamed method where you needed the second value

Reply:
thanks, done!